sdk/monitor/monitor-opentelemetry-exporter/samples/v1-beta/typescript/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@azure-samples/monitor-opentelemetry-exporter-ts-beta",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Azure Monitor Trace Exporter client library samples for TypeScript (Beta)",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "prebuild": "rimraf dist/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/azure-sdk-for-js.git",
+    "directory": "sdk/monitor/monitor-opentelemetry-exporter"
+  },
+  "keywords": [
+    "node",
+    "azure",
+    "monitor",
+    "javascript",
+    "applicationinsights",
+    "opentelemetry",
+    "exporter",
+    "cloud"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/monitor/monitor-opentelemetry-exporter",
+  "dependencies": {
+    "@azure/monitor-opentelemetry-exporter": "next",
+    "dotenv": "latest",
+    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/resources": "^1.2.0",
+    "@opentelemetry/semantic-conventions": "^1.2.0",
+    "@opentelemetry/sdk-trace-base": "^1.2.0",
+    "@opentelemetry/instrumentation": "^0.29.2",
+    "@opentelemetry/sdk-trace-node": "^1.2.0",
+    "@opentelemetry/instrumentation-http": "^0.29.2"
```

Comment:
@hectorhdzg Do you need to test the samples with the version update? Could you test them on your end if needed. The versions of these dependencies were updated - @opentelemetry/instrumentation and @opentelemetry/instrumentation-http.

Reply:
I can validate but I prefer to do it once instead of multiple times, if you can update all other OpenTelemetry dependencies would be great

Reply:
Yes all other Opentelemetry dependencies are also updated by `rush update` command. pnpm lock file shows the latest versions being used.